public/index.html

```diff
@@ -49,5 +49,17 @@
   To begin the development, run `npm start` or `yarn start`.
   To create a production bundle, use `npm run build` or `yarn build`.
 -->
+<script>
+  if('serviceWorker' in navigator) {
```

Comment:
I already have prettier installed, so I'm not sure why the formatting would be different.

Reply:
We found out why, this was a problem in the index.html file. It seems that <head> does not have a closing tag and thus prettier gets confused. Should this be changed, so that <head> gets a closing tag?